core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/kubernetes/KubernetesClient.scala

```diff
@@ -76,6 +76,7 @@ case class KubernetesClientConfig(timeouts: KubernetesClientTimeoutConfig,
                                   invokerAgent: KubernetesInvokerAgentConfig,
                                   userPodNodeAffinity: KubernetesInvokerNodeAffinity,
                                   portForwardingEnabled: Boolean,
+                                  actionNamespace: Option[String] = None,
```

Comment:
Would be helpful to have a comment in application.conf explaining this optional config

https://github.com/apache/openwhisk/blob/f3040925aa408a0b32dd4f542c815a638a797cc4/core/invoker/src/main/resources/application.conf#L65-L67

Reply:
Ah thanks, I hunted around for a markdown that describes the conf settings -- didn't think to peak there. Nice catch.